test/e2e/feature/feature.go

```diff
@@ -70,7 +70,13 @@ var (
 	// Marks tests that require coordinated leader election
 	CoordinatedLeaderElection = framework.WithFeature(framework.ValidFeatures.Add("CoordinatedLeaderElection"))
 
-	// TODO: document the feature (owning SIG, when to use this feature for a test)
+	// owning-sig: sig-node
+	// kep: https://kep.k8s.io/3570
+	// test-infra jobs:
+	// - https://testgrid.k8s.io/sig-node-kubelet#kubelet-serial-gce-e2e-cpu-manager
+	//
+	// This label is used for tests which need:
+	// - NUMA support available on the given node.
```

Comment:
This is not accurate, the label is needed to allow filtering e.g. run only  CPU Manager tests and not to target multi-NUMA systems.

Also want to clarify that not  all CPU manager tests need multi-NUMA environment, there are some constraints in terms on the number of CPUs and smtLevel on the test nodes.

Reply:
Maybe we need a separate feature tag for multi-numa.

@swatisehgal I think this PR is not making things worse. Would you agree? Should we merge it as is and iterate?

Reply:
@swatisehgal updated the description. please review it again.

Reply:
Yes, my comment has been addressed and I agree we can move ahead. 

I had done some work in tackling multi-NUMA in the past and need to determine the current state of things. Introducing a feature tag for multi-numa seems reasonable, can totally be done as a follow-up.

Reply:
/approve